XRTK-Core/Packages/com.xrtk.core/Runtime/Utilities/WindowsDevicePortal/DevicePortal.cs

```diff
@@ -871,7 +871,11 @@ private static async Task<Response> DevicePortalAuthorizationAsync(DeviceInfo ta
 
             await webRequest.SendWebRequest();
 
+#if UNITY_2020_1_OR_NEWER
+            if (webRequest.result == UnityWebRequest.Result.ConnectionError || webRequest.result == UnityWebRequest.Result.ProtocolError)
```

Comment:
This is may cause a conflict in the build tools refactoring in #780

Reply:
RACE!!!

Winner doesn't get to handle the merge conflicts :D

Reply:
Not a big deal. Even if we just take what Stephen has, I can easily restore these changes.